plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/config/CubridUserManager.java

```diff
@@ -16,30 +16,37 @@
  */
 package org.jkiss.dbeaver.ext.cubrid.ui.config;
```

Comment:
Please move CubridUserManager into plugin org.jkiss.dbeaver.ext.cubrid. It is non-UI manager.